src/server/utils/logging/fastifyPlugin.js

```diff
@@ -137,7 +138,7 @@ const logClientRequest = (request, reply) => {
   logger.info(configuredLog);
 };
 
-export const setConfigureRequestLog = (newConfigureRequestLog) => {
+export const setConfigureRequestLog = (newConfigureRequestLog = passThrough) => {
```

Comment:
why would somebody call `setConfigureRequestLog` without passing an argument?

Reply:
its called everytime during onModule load

Reply:
because it shouldn't be called without an argument

Reply:
It shouldn't but it is. I don't see the issue having a default to improve resiliency

Reply:
I won't block it but it's a bad practice. The function by starting with the word `set` lets the developer know that it "sets" something, and the data will always be expected (you cannot set something with no data, otherwise would be like a reset). That's the reason why all functions, like the `write` from `fs` will throw an error if data is not provided. If we want a set to be "safe" I would rename it to something like `safeSetConfigureRequestLog`, but, still, ideally, the user should always pass data